src/BackupManager/Factory/BackupManagerFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\Deploy\BackupManager\Factory;
+
+use BackupManager\Manager;
+
+class BackupManagerFactory implements BackupManagerFactoryInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function create($storageConfig, $dbConfig)
+    {
+        // Add all default filesystems.
+        $filesystemProvider = StorageFactory::create($storageConfig);
+
+        // Add all default databases.
+        $databaseProvider = DatabaseFactory::create($dbConfig);
+
+        // Add all default compressors.
+        $compressorProvider = CompressorFactory::create();
+
+        return new Manager($filesystemProvider, $databaseProvider, $compressorProvider);
```

Comment:
Because it was late yesterday and my head wasn't where it's supposed to be :-/ you're absolutely right.